ui/src/app/apps/app-details/app-details.component.ts

```diff
@@ -1,55 +1,195 @@
-import { Component, OnInit } from '@angular/core';
-import { Router, ActivatedRoute, Params } from '@angular/router';
-
-import { AppsService } from '../apps.service';
-import { ToastyService } from 'ng2-toasty';
-
-import { ApplicationType, DetailedAppRegistration } from '../../shared/model';
-
-import { Subscription } from 'rxjs/Subscription';
-import 'rxjs/add/observable/of';
+import {Component, OnInit} from '@angular/core';
+import {Router, ActivatedRoute, Params} from '@angular/router';
+import {AppsService} from '../apps.service';
+import {ToastyService} from 'ng2-toasty';
+import {ApplicationType, DetailedAppRegistration} from '../../shared/model';
+import {Subscription} from 'rxjs/Subscription';
+import {SharedAboutService} from '../../shared/services/shared-about.service';
+import {AppRegistration} from '../../shared/model/app-registration.model';
+import {Observable} from 'rxjs/Observable';
+import {FeatureInfo} from '../../shared/model/about/feature-info.model';
+import {AppVersionsComponent} from '../app-versions/app-versions.component';
+import {BsModalService} from 'ngx-bootstrap';
+import {SortParams} from '../../shared/components/shared.interface';
 
```

Comment:
import is missing for `combineLatest`, getting a runtime error. This works:

`import 'rxjs/add/observable/combineLatest';`

alternatively we may also embrace lettable operators and change as described here:

https://medium.com/@cartant/as-youve-used-it-combinelatest-isn-t-really-an-operator-it-s-an-observable-creator-bdab609dae58